edxlearndot/management/commands/update_learndot_enrolments.py

```diff
@@ -90,8 +103,11 @@ def handle(self, *args, **options):
         # for each mapped course, go through its enrollments, get the
         # course grade for each enrolled user, and if the user has passed,
         # update the Learndot enrolment
+
+        end_enrollments_date = dateparser.parse(options["end"])
+        start_enrolments_date = dateparser.parse(options["start"])
```

Comment:
Since these date strings don't have timezones by default, I'm getting a warning when I run this:

> RuntimeWarning: DateTimeField CourseEnrollment.created received a naive datetime (2021-04-21 06:41:53.098954) while time zone support is active.

This fixes that:

```diff
--- a/edxlearndot/management/commands/update_learndot_enrolments.py
+++ b/edxlearndot/management/commands/update_learndot_enrolments.py
@@ -10,6 +10,7 @@ import logging
 import sys
 
 import dateparser
+from django.conf import settings
 from django.core.management.base import BaseCommand
 
 from opaque_keys import InvalidKeyError
@@ -104,8 +105,9 @@ class Command(BaseCommand):
         # course grade for each enrolled user, and if the user has passed,
         # update the Learndot enrolment
 
-        end_enrollments_date = dateparser.parse(options["end"])
-        start_enrolments_date = dateparser.parse(options["start"])
+        date_settings = {'TIMEZONE': settings.TIME_ZONE, 'RETURN_AS_TIMEZONE_AWARE': True}
+        end_enrollments_date = dateparser.parse(options["end"], settings=date_settings)
+        start_enrolments_date = dateparser.parse(options["start"], settings=date_settings)
 
         for cm in course_mappings:
             try:
```

Reply:
nice, thank you. Improved